addons/search-profile/addon.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Search Profile",
+  "description": "Allows you to go to a users profile when you search their username",
+  "userscripts": {

```

Comment:
This must be an array of objects - it can't just be an object.

Reply:
Ah, my bad